### PR TITLE
Finish editing screen

### PR DESCRIPTION
**Pull request by Houra-Am**
### Motivation

If the user is the first one to create a mimic game, he will see the former page with "go to homepage" button. Otherwise, he will have the option to discover the other games.

### Changes

success.tsx modified

### Test

We need to create the games for the first time and change the user and create another game.

### CODE REVIEW

File: src/pages/creer-un-jeu/success.tsx [hunk 1]
```diff
@@ -7,30 +8,73 @@ import Button from '@mui/material/Button';
...
+      axiosLoggedRequest({
+        method: 'GET',
+        url: `/games/play${serializeToQueryUrl({
+          villageId: village.id,
+          type: GameType.MIMIC,
+        })}`,
+      }).then((response) => {
+        //if user is the first one to create the mimics, then "return to homepage" is rendered
+        if (!response.error && response.data === null) {
+          setFirstCreation(response.data as boolean);
```
Comment (DavidRobertAnsart) [inline src/pages/creer-un-jeu/success.tsx:40]: I don't understand the if condition and the type cast after. If `response.data` is `null`, then `response.data as boolean` is always `false` ?

**Suggestion:** should it be **not** null?
```typescript
if (!response.error && response.data !== null) {
    setFirstCreation(response.data as boolean);
}
```

File: src/pages/creer-un-jeu/success.tsx [hunk 2]
```diff
@@ -7,30 +8,73 @@ import Button from '@mui/material/Button';
...
+        } else {
+          setFirstCreation(true);
+        }
+      });
+    }
+  }, [firstCreation, axiosLoggedRequest, village]);
```
Comment (DavidRobertAnsart) [inline src/pages/creer-un-jeu/success.tsx:46]: Do you need the `firstCreation` in the useEffect dependancies? If the effect updates the `firstCreation` value, there could be an infinite loop.

**Suggestion:** remove it if it's not needed.

File: src/pages/creer-un-jeu/success.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (DavidRobertAnsart) [inline src/pages/creer-un-jeu/success.tsx:1]: This is not needed anymore?

File: src/pages/creer-un-jeu/success.tsx (hunk 1 above)
Comment (Houra-Am) [reply src/pages/creer-un-jeu/success.tsx]: Ok, I have applied the suggestion.

File: src/pages/creer-un-jeu/success.tsx (hunk 2 above)
Comment (Houra-Am) [reply src/pages/creer-un-jeu/success.tsx]: I removed the firstCreation  from the dependencies.

File: src/pages/creer-un-jeu/success.tsx (hunk 3 above)
Comment (Houra-Am) [reply src/pages/creer-un-jeu/success.tsx]: I removed the elsint no-console.

**Review by DavidRobertAnsart — APPROVED** [review]
Lgmt!

**Pull request merged**